recipes-core/packagegroups/packagegroup-ni-internal-deps.bb

```diff
@@ -39,3 +39,11 @@ RDEPENDS_${PN}_append_x64 = "\
 RDEPENDS_${PN} += "\
 	rdma-core \
 "
+
+# ni-flexrio-integratedio-libs, required for csi2serdesconfig
+# Contact: Michael Strain <michael.strain@ni.com>
```

Comment:
If you haven't already: ping Michael Strain and let him know that his name will be on this requirement.

Reply:
Done. He said it's "probably fine."